tests/unit/test_target.py

```diff
@@ -0,0 +1,175 @@
+# -*- coding: utf-8 -*-
+'''
+    :codeauthor: :email: `Mike Place <mp@saltstack.com>`
```

Comment:
Looks like this is a new file, so the `codeauthor` should be updated here.

Reply:
It's a new file for this branch but it was back ported from 2017.7.